apidoc/Titanium/Geolocation/Geolocation.yml

```diff
@@ -612,50 +594,6 @@ properties:
     permission: read-only
     platforms: [iphone, ipad]
 
-  - name: PROVIDER_GPS
```

Comment:
These providers are still present and can be used if `ti.playservices` is not included.